flake8diff/vcs/git.py

```diff
@@ -65,3 +69,7 @@ def changed_files(self):
         return filter(self.filter_file,
                       iter(_execute(' '.join(command))
                            .splitlines()))
+
+    def _is_git_repository(self):
+        return _execute(
+            '{vcs} status'.format(vcs=self.vcs), strict=True, log_errors=False)
```

Comment:
@netme I'm not convinced `git status` would give you any better context about the state of the repository. A check for the `.git` directory would certainly be faster for the same information you're getting here.  If we were calling something like `git verify` (which doesn't exist) then I could definitely agree.

Anyway, no need to change, this is fine for now. :+1: